charts/objecten/templates/configmap.yaml

```diff
@@ -33,6 +33,45 @@ data:
   DB_USER: {{ .Values.settings.database.username | toString | quote }}
   DB_PORT: {{ .Values.settings.database.port | toString | quote }}
   PGSSLMODE: {{ .Values.settings.database.sslmode | toString | quote }}
+  {{- if .Values.settings.database.objecttype_version_cache_timeout }}
+  OBJECTTYPE_VERSION_CACHE_TIMEOUT: {{ .Values.settings.database.objecttype_version_cache_timeout | quote }}
+  {{- end }}  
+  {{- with .Values.settings.database.db_pool }}
+  {{- if .enabled }}
```

Comment:
Is there a reason why you didn't include `DB_POOL_ENABLED`? Without this value, DB connection pooling will not be enabled by the application.